test/JDBC/upgrade/latest/schedule

```diff
@@ -89,6 +89,7 @@ BABEL-3249
 BABEL-383
 BABEL-405
 BABEL-PROCID
+BABEL-3360
```

Comment:
why only to latest schedule file? we should add it to all other upgrade sources like 13.4, 13.5 up to 14.3